src/theme-mode-script.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
I guess that I'll only hope to see an update on the documentation as well.

https://www.flowbite-react.com/docs/customize/dark-mode

Reply:
Good point, let me see what I can do.

Reply:
Docs will come out right before we release, so there is no blocker if any other fix touches docs.

Reply:
ack